src/main/java/com/aws/greengrass/clientdevices/auth/iot/dto/CertificateV1.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.iot.dto;
+
+import com.aws.greengrass.clientdevices.auth.api.DomainEvent;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
+@AllArgsConstructor
+public class CertificateV1 implements DomainEvent {
```

Comment:
Use past tense for event names, they should indicate an action that happened.

Reply:
Oh, lol. We copied this class from a domain event. It shouldn't implement this. Will remove